api/src/main/java/org/pmiops/workbench/db/dao/InstitutionUserInstructionsDao.java

```diff
@@ -10,5 +10,9 @@
 
   Optional<DbInstitutionUserInstructions> getByInstitution(final DbInstitution institution);
 
+  Optional<DbInstitutionUserInstructions> getByInstitution_ShortName(final String shortName);
```

Comment:
Since this PR was already pretty big, i was thinking of having a follow up PR to remove this and the getInstitutionUserInstructions methods thats using it